internal/plugin/connectors/http/generic/external_api.go

```diff
@@ -0,0 +1,75 @@
+package generichttp
+
+import (
+	"fmt"
+
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector/http"
+	"gopkg.in/yaml.v2"
```

Comment:
Needs sorting

Reply:
Do we have any sort of check style for sorting imports, or a means of adding one?